### PR TITLE
v6.0.0-alpha.14

### PR DESCRIPTION
**Pull request by LukasTy**
- [x] Compare the last tag with the branch upon which you want to release (`next` for the alpha / beta releases and `master` for the current stable version).
      To do so, use `yarn release:changelog` The options are the following:

  ```bash
  yarn release:changelog
     --githubToken   YOUR_GITHUB_TOKEN (needs "public_repo" permission)
     --lastRelease   The release to compare against (default: the last one)
     --release       The branch to release (default: master)
  ```

  You can also provide the github token by setting `process.env.GITHUB_TOKEN` variable.

  In case of a problem, another method to generate the changelog is available at the end of this page.

- [x] Clean the generated changelog, to match the format of [https://github.com/mui/mui-x/releases](https://github.com/mui/mui-x/releases).
- [x] Update the root `package.json`'s version
- [x] Update the versions of the other `package.json` files and of the dependencies with `yarn release:version`.
- [x] Manually fix any package version if it should not be bumped (i.e. `benchmark`, `eslint-plugin-material-ui`).
- [x] Open PR with changes and wait for review and green CI.
- [ ] Merge PR once CI is green, and it has been approved.

### Release the packages

- [ ] Checkout the last version of the working branch
- [ ] Make sure you have the latest dependencies installed: `yarn`.
- [ ] Build the packages: `yarn release:build`.
- [ ] Release the versions on npm: `yarn release:publish` (you need your 2FA device).
- [ ] Push the newly created tag: `yarn release:tag`.

### Publish the documentation

The documentation must be updated on the `docs-vX` branch (`docs-v4` for `v4.X` releases, `docs-v5` for `v5.X` releases, ...)

- [ ] Push the working branch on the documentation release branch to deploy the documentation with the latest changes.

```sh
git push upstream next:docs-next -f
```

You can follow the deployment process [on the Netlify Dashboard](https://app.netlify.com/sites/material-ui-x/deploys?filter=docs-v5)
Once deployed, it will be accessible at https://material-ui-x.netlify.app/ for the `docs-v5` deployment.

### Publish GitHub release

- [ ] After documentation is deployed, publish a new release on [GitHub releases page](https://github.com/mui/mui-x/releases)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,84 @@
...
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.0.0-alpha.14
+
+_Jan 5, 2023_
+
+We'd like to offer a big thanks to the 8 contributors who made this release possible. Here are some highlights ✨:
+
+- 📆 Add `SingleInputTimeRangeField` and `SingleInputDateTimeRangeField` components
+- 🚀 Use grid for modifying pickers layout
```
Comment (LukasTy) [inline CHANGELOG.md:13]: @mui/xgrid Do we have any other bigger changes worth highlighting?

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,84 @@
...
+- [DataGrid] Invert generic parameters order (#6874) @DanailH
+- [DataGrid] Remove legacy logic for `singleSelect` inside `GridFilterInputValue` (#7386) @m4theushw
+- [DataGrid] Remove remaining props from legacy editing API (#7381) @m4theushw
+- [DataGrid] Set default `GridCellParams['value']` type to `unknown` (#6959) @cherniavskii
+
+### `@mui/x-date-pickers@v6.0.0-alpha.14` / `@mui/x-date-pickers-pro@v6.0.0-alpha.14`
+
+#### Breaking changes
+
+- [fields] Rename the `input` slot of the fields to `textField` to avoid confusion (#7369) @flaviendelangle
```
Comment (LukasTy) [inline CHANGELOG.md:39]: @flaviendelangle Should we add some more info here?

File: packages/x-codemod/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-codemod",
-  "version": "6.0.0-alpha.13",
+  "version": "6.0.0-alpha.14",
```
Comment (LukasTy) [inline packages/x-codemod/package.json:3]: [This PR](https://github.com/mui/mui-x/pull/7233) triggered the release.

File: CHANGELOG.md (hunk 2 above)
Comment (flaviendelangle) [reply CHANGELOG.md:39]: Given the amount of people currently using the beta, I think it's fine :laughing: 
If you want to add an example, no problem of course

File: CHANGELOG.md (hunk 1 above)
Comment (flaviendelangle) [reply CHANGELOG.md:13]: #7272 is a big change, but people not already using the fields probably don't care

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,84 @@
...
+- [docs] Redirect translated pages (#7341) @cherniavskii
+- [docs] Reorganize v6 pickers migration guide (#7257) @flaviendelangle
+
+### Core
+
+- [core] Apply eslint rule for React component @oliviertassinari
+- [core] Apply title capitalization convention @oliviertassinari
+- [core] Fix the product license reference name (#7367) @oliviertassinari
+- [core] Order the slots alphabetically in the JSON files (#7349) @flaviendelangle
+- [core] Remove blanklines in _redirects @oliviertassinari
```
Comment (flaviendelangle) [inline CHANGELOG.md:78]:
```suggestion
- [core] Remove blanklines in `_redirects` @oliviertassinari
```

**Review by flaviendelangle — APPROVED** [review]
I did not see anything blocking, it's super clean :partying_face:

**Review by alexfauquette — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (LukasTy) [reply CHANGELOG.md]: I've added it as well as added mentions and PR refs to those significant changes.

**Pull request merged**